### PR TITLE
Makes Helmets Smaller

### PR DESCRIPTION
**Pull request by XSlayer300**
## About The Pull Request
Makes any kinds of helmets (including marine ones) smaller. There's no reason for it to be bulky other than the fact that it doesn't protect against leaping facehuggers anymore. It just annoys me because of that fact.

## Why It's Good For The Game
Portability and for RP reasons, as well as non-expected contact RP.

## Changelog
:cl:
tweak: Made helmets smaller in size, allowing them to be put on satchels and bags again. However, their internal storage space is reduced from 4 - 3.
/:cl:

### CODE REVIEW

File: code/modules/clothing/head/helmet.dm [hunk 1]
```diff
@@ -13,7 +13,7 @@
 	min_cold_protection_temperature = HELMET_MIN_COLD_PROTECTION_TEMPERATURE
 	max_heat_protection_temperature = HELMET_MAX_HEAT_PROTECTION_TEMPERATURE
 	siemens_coefficient = 0.7
-	w_class = WEIGHT_CLASS_BULKY
+	w_class = WEIGHT_CLASS_SMALL
```
Comment (spookydonut) [inline code/modules/clothing/head/helmet.dm]:
```suggestion
	w_class = WEIGHT_CLASS_NORMAL
```
unless you want them to fit in pouches lol

**Comment by Rohesie** [comment]
If we are decreasing the size of helmets, the contents they can carry should also be lowered.

File: code/modules/clothing/head/helmet.dm [hunk 2]
```diff
@@ -205,7 +205,7 @@
 		/obj/item/ammo_magazine/pistol)
```
Comment (Rohesie) [inline code/modules/clothing/head/helmet.dm]: Remove this and the SMG holding.

File: code/modules/clothing/head/helmet.dm [hunk 3]
```diff
@@ -196,7 +196,7 @@
 						/obj/item/clothing/glasses/mgoggles/prescription = "goggles")
 
 /obj/item/storage/internal/marinehelmet
-	storage_slots = 4
+	storage_slots = 3
```
Comment (Rohesie) [inline code/modules/clothing/head/helmet.dm]: Make it hold only 2, like before.

File: code/modules/clothing/head/helmet.dm [hunk 4]
```diff
@@ -196,16 +196,14 @@
...
 /obj/item/storage/internal/marinehelmet
-	storage_slots = 4
+	storage_slots = 2
 	max_w_class = 1
 	bypass_w_limit = list(
 		/obj/item/clothing/glasses,
-		/obj/item/reagent_containers/food/drinks/flask,
-		/obj/item/ammo_magazine/smg,
-		/obj/item/ammo_magazine/pistol)
+		/obj/item/reagent_container/food/drinks/flask)
```
Comment (Rohesie) [inline code/modules/clothing/head/helmet.dm]:
```suggestion
		/obj/item/reagent_containers/food/drinks/flask)
```

**Pull request merged**